### ISSUE TITLE
[SCREENREADER]: GIBCT® Colmery 107 - Locations table has invalid HTML, could be improved for assistive tech

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1640]
## Description

The school locations table doesn't announce itself properly to Safari + VoiceOver, and has some invalid HTML. It would be good to refactor this table for assistive device users. Screenshot and code sample below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a VoiceOver user, I want to hear the column header and cell announced properly.
* As a user, I do not want the visual layout to change when the HTML is refactored.

## Environment

* MacOS Mojave
* Safari
* VoiceOver
* https://staging.va.gov/gi-bill-comparison-tool/profile/11806124

## Possible Fixes (optional)

I tried out this snippet of HTML in Safari on my local machine with DevTools and it worked as expected. Validated it with HTML5 Validator Nu.

```html
<table>
  <caption>
    Below are locations for UNIVERSITY OF MISSISSIPPI. The housing estimates shown here are based on a full-time student taking in-person classes. Select a link to view a location and calculate the benefits you’d receive there.
  </caption>
  <thead>
    <tr>
      <th scope="col">School Name</th>
      <th scope="row">Location</th>
      <th scope="row">Estimated housing</th>
    </tr>
  </thead>
  <tbody>
    <tr class="main-row">
      <th scope="row">UNIVERSITY OF MISSISSIPPI (Main Campus)</th>
      <td class="location-cell">UNIVERSITY, MS 38677</td>
      <td>$1,437/mo</td>
    </tr>
    <tr class="extension-row">
      <th scope="row"><span>&nbsp;&nbsp;&nbsp;&nbsp;UNIVERSITY OF MISSISSIPPI BOONVILLE</span></th>
      <td class="location-cell">BOONVILLE, MS 38829</td>
      <td>$1,119/mo</td>
    </tr>
    <tr class="extension-row">
      <th scope="row"><span>&nbsp;&nbsp;&nbsp;&nbsp;UNIVERSITY OF MISSISSIPPI DESOTO SOUTHAVEN</span></th>
      <td class="location-cell">SOUTHAVEN, MS 38671</td>
      <td>$1,389/mo</td>
    </tr>
    <tr class="extension-row">
      <th scope="row"><span>&nbsp;&nbsp;&nbsp;&nbsp;UNIVERSITY OF MISSISSIPPI GRENADA</span></th>
      <td class="location-cell">GRENADA, MS 38901</td>
      <td>$1,143/mo</td>
    </tr>
    <tr class="extension-row">
      <th scope="row"><span>&nbsp;&nbsp;&nbsp;&nbsp;UNIVERSITY OF MISSISSIPPI TUPELO</span></th>
      <td class="location-cell">TUPELO, MS 38804</td>
      <td>$1,218/mo</td>
    </tr>
    <tr class="extension-row">
      <th scope="row"><span>&nbsp;&nbsp;&nbsp;&nbsp;UNIVERSITY OF MISSISSIPPI UMMC JACKSON</span></th>
      <td class="location-cell">JACKSON, MS 39216</td>
      <td>$1,353/mo</td>
    </tr>
  </tbody>
</table>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## PR
https://github.com/department-of-veterans-affairs/vets-website/pull/10712

## Screenshots or Trace Logs

![Screen Shot 2019-09-09 at 12.58.49 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/de3cd3ef-3bdd-42b2-933c-a04e481ccab7)

---
![Screen Shot 2019-09-09 at 1.03.27 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/0fb4c7f7-ee2f-40d9-bcf7-0ce2c0cc376b)

**Comment by NateJohnson-BAH** [comment]
PR link: https://github.com/department-of-veterans-affairs/vets-website/pull/10712

**Comment by delli-gatti-michael** [comment]
This table appears to be working with voiceover now. Animated gif showing what voiceover reads below:


![voiceover.gif](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/acbb8b06-ed81-4a36-abe1-df2cb61936c1)

**Comment by NateJohnson-BAH** [comment]
PR was merged as of 1400 on 16 Sep, 2019.

**Comment by 1Copenut** [comment]
@NateJohnson-BAH I verified this ticket with IE11 and Chrome with JAWS, and Firefox and Chrome with NVDA. All are reporting the table as expected. Moving to close.
